docs/statistics.md

```diff
@@ -0,0 +1,561 @@
+
+## byteRcvDrop
+
+Same as `byteRcvDropTotal`, but on the interval.
+
+## byteRcvUndecrypt
+
+Same as `byteRcvUndecryptTotal`, but on the interval.
+
+# Instant measurements
+
+The measurements effective at the time retrieved.
+
+## usPktSndPeriod
+
+Current minimum time interval between which consecutive packets are sent, in 
+microseconds. Sender only.
+
+Note that several sockets sharing one outgoing port use the same sending queue.
+They may have different pacing of the outgoing packets, but all the packets will
+be placed in the same sending queue, that may affect actual sending timing.
+
+`usPktSndPeriod` is the minimum time (sending period) that must be kept
+between two sent consecutively sent packets over the link used by SRT socket.
+It is not the EXACT time interval between
+two consecutive packets are sent. In the case where the time spent by the 
+application between two consecutive sendings exceeds `usPktSndPeriod`, the next 
+packet will be sent faster or even immediately to preserve the average sending rate.
+
+**Note**. Except for probing packets.
+
+## pktFlowWindow
+
+The maximum number of packets that can be "in flight". Sender only.
+See also [pktFlightSize](#pktFlightSize).
+
+The value represents the amount of free space
+on the peer receiver, stating that this socket represents the sender. When this
+value drops to zero, the next packet sent will be dropped by the receiver
+without processing. In **file mode** this may cause a slowdown of sending in
+order to wait until the receiver has more space available, after it
+eventually extracts the packets waiting in its receiver buffer; in **live
+mode** the receiver buffer contents should normally occupy not more than half
+of the buffer size (default 8192). If `pktFlowWindow` value is less than that
+and becomes even less in the next reports, it means that the receiver
+application on the peer side cannot process the incoming stream fast enough and
+this may lead do a dropped connection.
+
+
+## pktCongestionWindow
+
+Congestion window size, in number of packets. Sender only.
+
+Dynamically limits the maximum number of packets that can be in flight.
+Congestion control module dynamically changes the value.
+
+In **file mode**  this value starts at 16 and is increased to the number of reported
+acknowledged packets. Then is also updated based on the delivery rate, reported by the receiver.
+It represents the maximum number of packets that can be safely
+sent without causing network congestion. The higher this value is, the faster the
+packets can be sent. In **live mode** this field is not used.
+
+## pktFlightSize
+
+The number of packets in flight. Sender only.
+
+`pktFlightSize <= pktFlowWindow` and `pktFlightSize <= pktCongestionWindow`
+
+This is the distance 
+between the packet sequence number that was last reported by an ACK message and 
+the latest sequence number of the packet, that was sent (at the moment when the statistics
+are being read).
```

Comment:
the sequence number of the latest packet sent (at the moment when the statistics
are being read).